File: repos_estudiantes_TARQUI_primera_entrega/MURILLO_PEÑARANDA_MARIO_FABRICIO/Introduccion a POO/ejercicio1/codigo/Persona/src/persona/Persona.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package persona;

/**
 *
 * @author Fabricio
 */
/*
 * ENUNCIADO:
 * 1. Crea una clase Persona con nombre, edad y ciudad
 * a) Agrega un método para mostrar el saludo: "Hola, soy (nombre) de {ciudad}"
 * b) Crea tres personas y muestra su saludo
 * c) Agrega un método para verificar si es mayor de edad
 */
public class Persona {
    private String nombre;
    private int edad;
    private String ciudad;

    public Persona(String nombre, int edad, String ciudad) {
        this.nombre = nombre;
        this.edad = edad;
        this.ciudad = ciudad;
    }

    public void saludar() {
        System.out.println("Hola, soy " + nombre + " de " + ciudad);
    }

    public boolean esMayorDeEdad() {
        return edad >= 18;
    }

    public static void main(String[] args) {
        Persona p1 = new Persona("Ana", 25, "La Paz");
        Persona p2 = new Persona("Luis", 16, "Cochabamba");
        Persona p3 = new Persona("Maria", 30, "Santa Cruz");

        p1.saludar();
        p2.saludar();
        p3.saludar();

        System.out.println(p1.nombre + " es mayor: " + p1.esMayorDeEdad());
        System.out.println(p2.nombre + " es mayor: " + p2.esMayorDeEdad());
        System.out.println(p3.nombre + " es mayor: " + p3.esMayorDeEdad());
    }
}

File: repos_estudiantes_TARQUI_primera_entrega/QUISPE_MALDONADO_CINTHIA_JAZMIN/INTRODUCCION-POO/EJERCICIO-1/Persona.py
class Persona:
    def __init__(self, nombre, edad, ciudad):
        self.nombre = nombre
        self.edad = edad
        self.ciudad = ciudad

    def saludo(self):
        print("Hola, soy " , self.nombre , " de " , self.ciudad)

    def esMayor(self):
        if self.edad >= 18:
            print("--Sí es mayor de edad")
        else:
            print("--No es mayor de edad")
p1 = Persona("Ana", 22, "Perú")
p2 = Persona("Miguel", 17, "Bolivia")
p3 = Persona("Felipe", 25, "Colombia")

p1.saludo()
p1.esMayor()

p2.saludo()
p2.esMayor()

p3.saludo()
p3.esMayor()
